core/src/test/scala/io/delta/tables/DeltaTableBuilderSuite.scala

```diff
@@ -228,6 +228,16 @@ class DeltaTableBuilderSuite extends QueryTest with SharedSparkSession with Delt
     }
   }
 
+  test("test addColumn using columnBuilder, without dataType") {
+    val e = intercept[AnalysisException] {
+      DeltaTable.columnBuilder("value")
+        .generatedAlwaysAs("true")
+        . nullable(true)
+        .build()
+    }
+    assert(e.getMessage.equals(s"The data type of column value is not provided"))
```

Comment:
nit: Use `==` for `equals`

Reply:
replaced equals with == and removed the brackets '()' is this okay ?